### ISSUE TITLE
Dependency change causes build to fail on travis

### ISSUE THREAD

**Issue by rcoh** [SumoLogic/elasticsearch-client#20]
```
[ERROR] Failed to execute goal on project elasticsearch-core: Could not resolve dependencies for project com.sumologic.elasticsearch:elasticsearch-core:jar:1.0.12-SNAPSHOT: Could not find artifact org.json4s:json4s-native_2.11:jar:3.4.0-SUMO in central (http://repo.maven.apache.org/maven2) -> [Help 1]
```

https://travis-ci.org/SumoLogic/elasticsearch-client/builds/124351424 cc @jakozaur

**Comment by davidandrzej** [comment]
@williamleferrand we should be depending on a publicly available `json4s` version?

**Comment by CCheSumo** [comment]
@davidandrzej @rcoh  I believe https://github.com/SumoLogic/elasticsearch-client/pull/18 should fix it. Do we agree we should merge it?

**Comment by rcoh** [comment]
+1

**Comment by CCheSumo** [comment]
Merged #18
